packages/components/src/divider/stories/index.tsx

```diff
@@ -23,18 +24,26 @@ export default {
 		marginEnd: {
 			control: { type: 'number' },
 		},
+		// TODO: This is a reakit internal prop. We can hide it from the props table like this
+		// if we don't want to expose it, but should we rather omit it at the TypeScript level?
+		unstable_system: {
+			table: { disable: true },
+		},
 	},
-};
+	parameters: {
+		controls: { expanded: true },
+	},
+} as ComponentMeta< typeof Divider >;
 
-const HorizontalTemplate = ( args ) => (
+const HorizontalTemplate: ComponentStory< typeof Divider > = ( args ) => (
```

Comment:
These `Component*` typings add a bit to the boilerplate, but they are what prevent mistakes in the `argTypes` and `args` objects.